Dockerfile

```diff
@@ -21,7 +21,21 @@ RUN apt-get update && \
     python3-dev \
     python3-pip \
     python3-setuptools \
-    python3-wheel && \
+    python3-wheel \
+    libairspy-dev \
+    libairspyhf-dev \
+    libavahi-client-dev \
+    libbsd-dev \
+    libfftw3-dev \
+    libhackrf-dev \
+    libiniparser-dev \
+    libncurses5-dev \
+    libopus-dev \
+    librtlsdr-dev \
+    portaudio19-dev \
+    libasound2-dev \
+    uuid-dev \
+    rsync && \
```

Comment:
I suspect you could pare this list down to just `libavahi-client-dev libbsd-dev libfftw3-dev libiniparser-dev libopus-dev` if you only build the targets you need below, i.e. `make -f Makefile.linux tune powers pcmcat`. This might speed up the build a bit.

Reply:
You are correct! Removing those extra packages and better defining the make command worked as expected.